discord/ui/view.py

```diff
@@ -531,29 +618,43 @@ async def wait(self) -> bool:
 
     def disable_all_items(self, *, exclusions: list[Item] | None = None) -> None:
         """
-        Disables all items in the view.
+        Disables all buttons and select menus in the view.
 
         Parameters
         ----------
         exclusions: Optional[List[:class:`Item`]]
             A list of items in `self.children` to not disable from the view.
         """
         for child in self.children:
-            if exclusions is None or child not in exclusions:
+            if hasattr(child, "disabled") and (
+                exclusions is None or child not in exclusions
+            ):
                 child.disabled = True
+            if hasattr(child, "items"):
+                child.disable_all_items(exclusions=exclusions)
+        return self
 
     def enable_all_items(self, *, exclusions: list[Item] | None = None) -> None:
         """
-        Enables all items in the view.
+        Enables all buttons and select menus in the view.
 
         Parameters
         ----------
         exclusions: Optional[List[:class:`Item`]]
             A list of items in `self.children` to not enable from the view.
         """
         for child in self.children:
-            if exclusions is None or child not in exclusions:
+            if hasattr(child, "disabled") and (
+                exclusions is None or child not in exclusions
+            ):
                 child.disabled = False
+            if hasattr(child, "items"):
+                child.enable_all_items(exclusions=exclusions)
+        return self
+
+    def copy_text(self) -> str:
+        """Returns the text of all :class:`~discord.ui.TextDisplay` items in this View. Equivalent to the `Copy Text` option on Discord clients."""
```

Comment:
```suggestion
        """Returns the text of all :class:`~discord.ui.TextDisplay` items in this View.
        Equivalent to the `Copy Text` option on Discord clients."""
```
Not sure if this is the right format